### PR TITLE
Translate pol.json and Default Settings SoD_SP.ini

### PR DESCRIPTION
**Pull request by pawel54321**
[+] Completly Translation for language Polish.
[+] Repaired polish characters.
[+] (Optional) Example Settings for language Polish.

### CODE REVIEW

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 1]
```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
 # Цвет координат курсора мыши на Карте Приключений
 # Die Koordinaten des Mauszeigers während man auf der Abenteuerkarte ist
 # Συντεταγμένες του κέρσορα του ποντικιού όταν βρίσκεστε στον χάρτη περιπέτειας (Adventure Map)
-AdventureCoord = H3Cyan
+AdventureCoord = Regular
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:34]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 2]
```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
...
 
 # The color of hit points of the creature under the mouse cursor while in combat
 # Цвет здоровья стека существ при наведении курсора мыши во время боя
 # Die Farbe der Lebenspunkte der Kreatur unter dem Mauszeiger während der Kampf
 # Το χρώμα των πόντων ζωής του πλάσματος κάτω από τον κέρσορα του ποντικιού κατά τη διάρκεια της μάχης
-HitPoints = H3Orange
+HitPoints = Regular
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:40]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 3]
```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
...
 
 # The color used when all targetted creatures will perish / or be resurrected
 # Цвет, означающий, что все целевого стека существ погибнут или будут воскрешены
 # Die benutzte Farbe wann alle gezielten Kreaturen sterben werden / oder sie auferstanden werden werden
 # Το χρώμα που χρησιμοποιείται όταν όλα τα στοχευμένα πλάσματα θα πεθάνουν / ή θα αναστηθούν# Το χρώμα που χρησιμοποιείται όταν μόνο ένα τμήμα των στοχευμένων πλασμάτων θα πεθάνει / ή θα αναστηθεί
-KillAll = H3Green
+KillAll = Regular
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:46]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 4]
```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
...
 
 # The color used when only a portion of targetted creatures will perish / or be resurrected
 # Цвет, означающий, что только часть целевого стека существ погибнет или будет воскрешена
 # Die benutzte Farbe wann nur ein Teil der gezielten Kreaturen sterben wird / oder ein Teil auferstanden werden wird
 # Το χρώμα που χρησιμοποιείται όταν μόνο ένα τμήμα των στοχευμένων πλασμάτων θα πεθάνει / ή θα αναστηθεί
-KillSome = H3Red
+KillSome = Regular
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:52]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 5]
```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
...
 
 # The color of move sequences while in combat
 # Цвет показателя очередности хода стеков в бою
 # Die Farbe der Bewegungsreihenfolge während der Kampf
 # Το χρώμα της ακολουθίας κινήσεων κατά τη διάρκεια της μάχης
-MoveSequence = H3Yellow
+MoveSequence = Regular
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:58]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 6]
```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
...
 
 # The color of coordinates under the mouse cursor while in combat
 # Цвет координат под курсором мыши во время боя
 # Die Farbe der Koordinaten unter dem Mauszeiger während der Kampf
 # Το χρώμα των συντεταγμένων κάτω από τον κέρσορα κατά τη διάρκεια της μάχης
-BattleCoord = Black
+BattleCoord = Regular
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:64]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 7]
```diff
@@ -94,37 +94,37 @@ BattleCoord = Black
 # Es deaktiviert „den Monat der SEUCHE!“
 # Απενεργοποιεί «τον μήνα της ΠΑΝΟΥΚΛΑΣ!»
 # Wyłącza "miesiąc PLAGI!".
-DisablePlague = 1
+DisablePlague = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:97]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 8]
```diff
@@ -94,37 +94,37 @@ BattleCoord = Black
...
-# Es deaktiviert die 12 Doppeltwachstumsmonate und die laichenden Monster auf der Karte 
+# Es deaktiviert die 12 Doppeltwachstumsmonate und die laichenden Monster auf der Karte
 # ANMERKUNG: Es streicht *NICHT* „den Monat des Imps“
 # Απενεργοποιεί τους 12 μήνες διπλής ανάπτυξης και την εμφάνιση τεράτων στον χάρτη
-# ΣΗΜΕΙΩΣΗ: *ΔΕΝ* αφαιρεί τον «Μήνα του Ιμπ» 
+# ΣΗΜΕΙΩΣΗ: *ΔΕΝ* αφαιρεί τον «Μήνα του Ιμπ»
 # Wyłącza miesiące 12 stworzeń, które dublują swoją populację i pojawiają się na mapie.
 # UWAGA: *NIE* usuwa "Miesiąca Impa".
-DisableDoubleGrowth = 1
+DisableDoubleGrowth = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:109]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 9]
```diff
@@ -94,37 +94,37 @@ BattleCoord = Black
...
 
 # Disables the random +5 Weekly Growth
 # Отключение случайного +5 бонуса недельного прироста
-# Es deaktiviert den zufälligen +5 wöchentlichen Wachstum 
+# Es deaktiviert den zufälligen +5 wöchentlichen Wachstum
 # Απενεργοποιεί την τυχαία +5 εβδομαδιαία ανάπτυξη
 # Wyłącza losowe tygodnie z bonusem +5 do przyrostu.
-DisableWeeklyGrowth = 1
+DisableWeeklyGrowth = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:116]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 10]
```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
 # Es zeigt die Rohstoffquellen (Gold, Holz, usw.) im 123,456X Format. X ist eine Zehnerpotenz (z.B. 123,456,789 wird sich als 123,456K zeigen)
 # Εμφανίζει τους φυσικούς πόρους (χρυσό, ξύλο κλπ.) χρησιμοποιώντας τη μορφή 123,456Χ, όπου Χ είναι μια δύναμη του δέκα (π.χ. το 123,456,789 θα εμφανίζεται ως 123,456K)
 # Pokazuje ilość surowców (złoto, drewno, itp.) używając formatowania 123,456X , gdzie X jest odpowiednim przedrostkiem liczby 10 (np. ilość 123,456,789 zostanie wyświetlona jako 123,456K).
-ResourcesFormat = 1
+ResourcesFormat = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:184]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 11]
```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
...
 
 # Combat textbox remains open after right-clicking on an opponent (lets you see some spells duration etc.)
 # Изменить поведение информационного окна при клике правой кнопкой мыши на вражеском существе (окно остается открытым)
 # Die Kampf-Textbox bleibt geöffnet nach Rechtsklick auf einen Gegner (Es erlaubt dem Spieler die Dauer einiger Zaubersprüchen zu sehen usw.)
-# Το πλαίσιο κειμένου στη μάχη παραμένει ανοικτό μετά το δεξί κλικ σε αντίπαλο (σας επιτρέπει να δείτε τη διάρκεια που έχουν ορισμένα ξόρκια κλπ.)  
+# Το πλαίσιο κειμένου στη μάχη παραμένει ανοικτό μετά το δεξί κλικ σε αντίπαλο (σας επιτρέπει να δείτε τη διάρκεια που έχουν ορισμένα ξόρκια κλπ.)
 # W trakcie walki okna informacyjne po kliknięciu prawym klawiszem myszki są dostępne (nie znikają) także dla jednostek wroga (umożliwią np. sprawdzenie długości działania niektórych zaklęć).
-CombatRMBTextbox = 1
+CombatRMBTextbox = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:191]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 12]
```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
...
 
 # Shows alpha-numeric combat coordinates when you mouse over various hexes during combat
 # При наведении и перемещении мыши отображать координаты гексов поля боя
 # Es zeigt alphanumerische Kampf-Koordinaten, wann man die Maus über verschiedene Hexadezimalschreibweisen während der Kampf bewegt
 # Εμφανίζει αλφαριθμητικές συντεταγμένες μάχης όταν μετακινείτε τον κέρσορα πάνω από διάφορα τετραγωνάκια κατά τη διάρκεια της μάχης
 # Pokazuje w formie alfanumerycznej koordynaty walki gdy podczas potyczki przesuwasz wskaźnik myszki nad heksami.
-CombatCoordinates = 1
+CombatCoordinates = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:198]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 13]
```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
...
 
 # Show creature's hitpoints when you mouse over them in combat
 # При наведении мыши на существо отображать очки его здоровья в активном логе битвы
 # Es zeigt die Lebenspunkte einer Kreatur, wann man die Maus über sie während der Kampf bewegt
 # Εμφανίζει τους πόντους ζωής ενός πλάσματος όταν περνάτε τον κέρσορα από πάνω του στη διάρκεια της μάχης
 # Pokazuje pozostałe punkty życia stworzenia, gdy podczas walki najedziesz na niego kursorem.
-ShowHitPoints = 1
+ShowHitPoints = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:205]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 14]
```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
...
+# ПРИМЕЧАНИЕ: опция экспериментальная и не откалиброванная (возможны неточности)
 # Es zeigt die Bewegungsreihenfolge der Kreatur, wann man die Maus über sie während der Kampf hält
 # ANMELDUNG: Es ist noch vorläufig und nicht absolut genau
 # Εμφανίζει τη σειρά κίνησης του πλάσματος όταν περνάτε το ποντίκι από πάνω του στη διάρκεια της μάχης
-# ΣΗΜΕΙΩΣΗ: Είναι ακόμα σε πειραματικό στάδιο και δεν είναι ακόμα απολύτως ακριβές 
+# ΣΗΜΕΙΩΣΗ: Είναι ακόμα σε πειραματικό στάδιο και δεν είναι ακόμα απολύτως ακριβές
 # Podczas walki tuż przy kursorze pokazuje kolejność ruchu stworzenia.
 # UWAGA: do wciąż eksperymentalna opcja i nie do końca działająca
-ShowMoveSequence = 1
+ShowMoveSequence = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:217]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 15]
```diff
@@ -238,7 +238,7 @@ SpellDetails = 1
 # Показывать координаты (x,y,z) курсора мыши на Карте Приключений
 # Es zeigt die Koordinaten des Mauszeigers (x,y,z) auf der Abenteuerkarte
 # Δείχνει τις συντεταγμένες του κέρσορα του ποντικιού (x,y,z) στον χάρτη περιπέτειας (Adventure Map)
-AdventureCoordinates = 1
+AdventureCoordinates = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:241]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 16]
```diff
@@ -250,28 +250,28 @@ RealKills = 1
 # Во время боя в левой части экрана будет отображаться интерактивная панель быстрого доступа к наиболее часто используемым заклинаниям.
 # Die linke Seite des Kampf-Fensters zeigt eine interaktive Tafel mit Ihren festgesetzten Schnellzaubersprüchen an und erlaubt Anpassung.
 # Το αριστερό τμήμα της οθόνης μάχης εμφανίζει έναν πίνακα με τα γρήγορα ξόρκια που έχετε επιλέξει και επιτρέπει εξατομίκευση.
-QuickSpellPanel = 1
+QuickSpellPanel = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:253]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 17]
```diff
@@ -250,28 +250,28 @@ RealKills = 1
...
 
 # The right hand side of the combat window displays a panel with the upcoming move order of creatures.
 # Во время боя в правой части экрана будет отображаться панель очередности хода существ.
 # Die rechte Seite des Kampf-Fensters zeigt eine Tafel mit der bevorstehender Bewegungsreihenfolge der Kreaturen an.
 # Το δεξιό τμήμα της οθόνης μάχης εμφανίζει έναν πίνακα με την επερχόμενη σειρά κίνησης των πλασμάτων.
-BattleQueuePanel = 1
+BattleQueuePanel = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:259]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 18]
```diff
@@ -306,7 +306,7 @@ EventBug = 1
 # Показ миникарты во время хода ИИ, при условии выключенной опции 'Не показывать движение врага'.
 # Die Minikarte wird, während die künstliche Intelligenz spielt, unter der Bedingung, dass die Bewegung der künstlicher Intelligenz nicht als „Keine Anzeige der Bewegung der Gegner“, angezeigt.
 # Ο μικρός χάρτης εμφανίζεται όταν είναι η σειρά να παίξει η τεχνητή νοημοσύνη, υπό την προϋπόθεση ότι η κίνηση της τεχνητής νοημοσύνης δεν έχει οριστεί στο «Να μην εμφανίζεται η κίνηση των εχθρών».
-AIMinimap = 1
+AIMinimap = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:309]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 19]
```diff
@@ -325,9 +325,9 @@ FastBattleBugs = 1
 
 # Naga Banks provide Sulfur instead of Gems. Requires New Game to take effect.
 # Сера в Нага Банках вместо драгоценных камней. Изменения вступят в силу только в новой игре.
-# Naga Banks stellen Schwefel statt Edelsteins zur Verfügung. Neus Spiel wird gebraucht um diese Änderung wirksam zu werden. 
+# Naga Banks stellen Schwefel statt Edelsteins zur Verfügung. Neus Spiel wird gebraucht um diese Änderung wirksam zu werden.
 # Τα Naga banks παρέχουν θειάφι αντί πολύτιμων λίθων. Απαιτείται νέο παιχνίδι για να ισχύσει η αλλαγή.
-NagaBank = 0
+NagaBank = 1
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:330]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 20]
```diff
@@ -357,13 +357,13 @@ HeroFusion = 1
 # Волшебное Зеркало работает и против войск ИИ игрока
 # Der Magische Spiegel des Spielers ist berücksichtigt, wenn die künstliche Intelligenz einen Zauberspruch wirkt
 # Το ξόρκι {Magic Mirror (Μαγικός Kαθρέφτης)} λαμβάνεται υπ' όψιν όταν η Τεχνητή Νοημοσύνη χρησιμοποιεί ένα ξόρκι
-AIMagicMirror = 1
+AIMagicMirror = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:360]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini [hunk 21]
```diff
@@ -357,13 +357,13 @@ HeroFusion = 1
...
 
 # Enchanters will correctly consider all valid creatures on battlefield to cast their automatic spells
 # Автоматическое колдовство Чародеями будет производить проверку возможности каста заклинания на всю армию целевой стороны, а не только на первый стек.
 # Die Enchanters werden alle gültigen Kreaturen auf dem Schlachtfeld richtig berücksichtigen um ihre automatischen Zaubersprόche zu wirken
 # Οι Enchanters θα λαμβάνουν ορθά υπ' όψιν όλα τα έγκυρα πλάσματα στο πεδίο της μάχης για να εκτελέσουν τα αυτόματα ξόρκια τους
-EnchantersFix = 1
+EnchantersFix = 0
```
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:366]: The default ini options should not be modified, just the comments explaining what they do.

**Review by RoseKavalier — CHANGES_REQUESTED** [review]
Language looks very well, fantastic job - must have taken quite a while! Other translators have been doing this work over a few years so doing that in a few days is 👍 

As you can see from the copy-pasted comments, the default ini options should not be modified, that's the one thing missing before approval for merging.

Once again, good job!

**Comment by pawel54321** [comment]
I renamed.

Thank you very much for great project ;).

**Review by RoseKavalier — APPROVED** [review]
Looks good!

**Pull request merged**